pkg/providers/discoverers/cloud_foundry/provider_test.go

```
@@ -0,0 +1,1103 @@
+
+	"github.com/cloudfoundry/go-cfclient/v3/config"
+	"github.com/cloudfoundry/go-cfclient/v3/testutil"
+	cfTypes "github.com/konveyor/asset-generation/internal/models"
+	pTypes "github.com/konveyor/asset-generation/pkg/providers/types/provider"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	goCFClientTemplateURL = "git::https://github.com/cloudfoundry/go-cfclient.git//testutil/template"
```

golangci-lint (1.64.8)

const `goCFClientTemplateURL` is unused

(unused)